### DISCUSSION TITLE
Quarkus upgrade from 2.9.2.Final -> to 2.16.2.Final is causing '403 CORS Rejected - Invalid origin'

### DISCUSSION THREAD

**Discussion by kirankolli-addepar** [quarkusio/quarkus#31164] [Q&A]
I have an existing quarkus application with RestEasy Reactive , agroal and a few more extensions.
As soon as I upgraded from 2.9.2.Final to 2.16.2.Final all my Rest Resources started rejecting requests with HTTP code 403 CORS Rejected - Invalid origin.

I have set the below properties : 
quarkus.http.cors=true
quarkus.http.cors.origins=http://([a-z0-9\\.\\:\\-_]*)

Note - I have also tried by removing the 'quarkus.http.cors.origins' property but that did not help.

The requests go with the Http Header : 
Origin: http://localhost: <some port>

What should I do to resolve this issue ?
Tagging experts : @radcortez

**Comment by sberyozkin** [comment]
@kirankolli-addepar `http://localhost/:` without port does not seem like a valid Origin value. If it is then either adding a wildcard, see the migration guide, https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.16#no-wildcard-origin-support-by-default-for-cors-filter, or, indeed, a specific origin value as you have tried to, it is a better solution.
FYI, for the configured origins be treated as regexpressions they have to start/end with `/` - it is in JavaDocs

HTH
